lib/directives/tooltip.js

```diff
@@ -79,7 +79,7 @@ function generateId() {
 function compileHtml(str) {
     let div = document.createElement('div');
     div.innerHTML = String(str || '').trim();
-    const node = div.firstElementChild ? div.removeChild(div.FirstElementChild) : null;
+    const node = div.firstElementChild ? div.removeChild(div.firstElementChild) : null;
```

Comment:
My point is that this line isn't needed at all, as this var at functions scope and will be handles by GC after function ends

Reply:
I'd refactor whole func to:
```
// Compile a string of HTML and return the DOM element
function compileHtml(str) {
    let div = document.createElement('div');
    div.innerHTML = String(str || '').trim();
    return div.firstElementChild;
}
```

Reply:
But I need to remove the compiled template from the DIV, otherwise it still has a `parentElement` (which is the outer holder `div`).

parentElement is used to detect if the tooltip is still in the document DOM.